src/components/tl/ucp/alltoall/alltoall_onesided.c

```diff
@@ -61,4 +71,5 @@ void ucc_tl_ucp_alltoall_onesided_progress(ucc_coll_task_t *ctask)
 
     pSync[0]           = 0;
     task->super.status = UCC_OK;
+    ucc_tl_ucp_coll_dynamic_segment_finalize(task);
```

Comment:
Now that this function returns an error code, should we handle it ?

Reply:
i've changed this so the error code will be passed to the user as there would be something significantly wrong in order to fail to unmap.